### PR TITLE
Junit setup/test

### PR DESCRIPTION
**Pull request by Zerocreative0815**
added the junit setup in the pom and a test for helloworld  #4

### CODE REVIEW

**Comment by Zerocreative0815** [comment]
@nicolaiparlog

File: pom.xml [hunk 1]
```diff
@@ -61,6 +69,10 @@
                         </goals>
                     </execution>
                     <execution>
+                        <id>compile</id>
+                        <goals> <goal>compile</goal> </goals>
+                    </execution>
+                    <execution>
```
Comment (nipafx) [inline pom.xml]: The four lines here duplicate what you already have aboce them and leads to a failing build. You have to remove them.

:bulb: Always run the build locally to make sure everything works as expected.

File: src/test/kotlin/HelloWorldtests/HelloWorldtest.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package HelloWorldtests
+
+import org.junit.Test
+
+class HelloWorldtest{
```
Comment (nipafx) [inline src/test/kotlin/HelloWorldtests/HelloWorldtest.kt]: The class (and the file) has to be called `HelloWorldTest` (note the upper case `T`) for Maven to pick it up. It worked when you see _Hello from the otherside!!_ on the command line.

**Review by nipafx — CHANGES_REQUESTED** [review]
Almost there... :smiley:

**Comment by Zerocreative0815** [comment]
Closes #4

**Comment by nipafx** [comment]
:bulb: "Closes ..."-Hinweise müssen in den ersten Kommentar des PUR, sonst funktionieren sie nicht.

**Pull request merged**